.travis.yml

```diff
@@ -3,7 +3,7 @@ language: bash
 services:
   - docker
 before_script:
-  - docker build --tag keboola/docker-jupyter .
+  - docker-compose build
```

Comment:
why?

Reply:
I changed to the compose to support the arg, but yeah, that's not necessary if it'll be static